src/cryptography/utils.py

```diff
@@ -101,12 +101,8 @@ def verify_interface(iface, klass):
             )
 
 
-if sys.version_info >= (2, 7):
-    def bit_length(x):
-        return x.bit_length()
-else:
-    def bit_length(x):
-        return len(bin(x)) - (2 + (x <= 0))
+def bit_length(x):
+    return x.bit_length()
```

Comment:
We should just inline this into all the callers now; in this PR or a seperate one?

Reply:
This is effectively public API (I've seen other projects using this). If we want to get rid of it I think we need a deprecation cycle.

Reply:
We should probably at least stop using it ourselves; I'm ok with keeping it around.